docs/_includes/default.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>{{ page.title }}</title>
+    <link
+      rel="preload"
+      href="https://opencollective.com/static/images/user.svg"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="prefetch"
+      href="https://www.google-analytics.com/analytics.js"
+      crossorigin="anonymous"
+    />
```

Comment:
Why are we tracking?

Reply:
it's useful to have some metrics, otherwise we have no idea how many hits we're getting or where people are from (where to focus future i18n efforts..)